### DISCUSSION TITLE
请问作者大大mediatrack的Token

### DISCUSSION THREAD

**Discussion by dairan915** [AlistGo/alist#507] [Q&A]
token七天失效是不是意味着获取的直链有效期只有七天？还是说七天后人为修改配置里的token？有没有自动更新token的方法呢？不懂望大佬讲讲，嘿嘿🤭

**Comment by xhofe** [comment]
人为修改token 它不提供refreshtoken 且登录有验证码，能够破解验证码的话也可以直接用账号密码。

**Reply by dairan915** [reply]
好滴好滴
